cosmo_tester/framework/util.py

```diff
@@ -253,3 +280,10 @@ def _get_parent_obj_prop_name_by_path(self, prop_path):
     @staticmethod
     def _raise_illegal(prop_path):
         raise RuntimeError('illegal path: {0}'.format(prop_path))
+
+
+def generate_password():
+    length = 13
+    chars = string.ascii_letters + string.digits + '!@#$%^&*()'
+    random.seed = (os.urandom(1024))
```

Comment:
not sure what the added value for changing the seed here is?
It won't be any more random than it already is